.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,24 @@
+
+name: 'govulncheck'
+on:
+  ## Nightly
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
This is UTC IIUC.
It may be a good idea to add some fudge factor here. It does not necessarily need to run exactly at midnight UTC.

Reply:
It can be anything. 00:00 UTC is as good a time as any.